Moviebuff/LaunchScreen.storyboard

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm guessing you added this at some point and then deleted from your Xcode project, but it's still on disk and got committed into git! But I don't think you need this unless I'm missing something.